crates/plugin_css/src/lib.rs

```diff
@@ -192,26 +192,42 @@ impl Plugin for FarmPluginCss {
     -99
   }
 
-  /// Just load the cache, if the cache is invalidated, it will be reset when transform.
-  fn plugin_cache_loaded(
+  fn build_start(
     &self,
-    cache: &Vec<u8>,
-    _context: &Arc<CompilationContext>,
+    context: &Arc<CompilationContext>,
   ) -> farmfe_core::error::Result<Option<()>> {
-    let cache = deserialize!(cache, CssModulesCache);
-    let mut content_map = self.content_map.lock();
-
-    for (k, v) in cache.content_map {
-      content_map.insert(k, v);
+    if context.cache_manager.enable {
+      if let Some(cache) = context
+        .cache_manager
```

Comment:
This way is the same as `plugin_cache_load`, we want the cache key control by the module instead of the plugin, so we may need to add a API like `context.cache_manager.add_module_extra_info` the these info will be appended  when write the module cache.

When reading cache, we can use `handle_persistent_cached_module` hook to restore the cache back to the plugin